src/openrct2-ui/windows/Sign.cpp

```diff
@@ -54,121 +56,112 @@ static rct_widget window_sign_widgets[] = {
         { WWT_COLOURBTN, 1, 17,     28,         WH - 16,    WH - 5,     0xFFFFFFFF,     STR_SELECT_TEXT_COLOUR_TIP },       // Text colour
         { WIDGETS_END },
 };
+// clang-format on
 
-static void window_sign_mouseup(rct_window *w, rct_widgetindex widgetIndex);
-static void window_sign_mousedown(rct_window *w, rct_widgetindex widgetIndex, rct_widget* widget);
-static void window_sign_dropdown(rct_window *w, rct_widgetindex widgetIndex, sint32 dropdownIndex);
-static void window_sign_textinput(rct_window *w, rct_widgetindex widgetIndex, char *text);
-static void window_sign_viewport_rotate(rct_window *w);
-static void window_sign_invalidate(rct_window *w);
-static void window_sign_paint(rct_window *w, rct_drawpixelinfo *dpi);
-
+static void window_sign_mouseup(rct_window * w, rct_widgetindex widgetIndex);
+static void window_sign_mousedown(rct_window * w, rct_widgetindex widgetIndex, rct_widget * widget);
+static void window_sign_dropdown(rct_window * w, rct_widgetindex widgetIndex, sint32 dropdownIndex);
+static void window_sign_textinput(rct_window * w, rct_widgetindex widgetIndex, char * text);
+static void window_sign_viewport_rotate(rct_window * w);
+static void window_sign_invalidate(rct_window * w);
+static void window_sign_paint(rct_window * w, rct_drawpixelinfo * dpi);
 
 // 0x98E44C
-static rct_window_event_list window_sign_events = {
-    nullptr,
-    window_sign_mouseup,
-    nullptr,
-    window_sign_mousedown,
-    window_sign_dropdown,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    window_sign_textinput,
-    window_sign_viewport_rotate,
-    nullptr,
-    nullptr,
-    nullptr,
-    nullptr,
-    window_sign_invalidate,
-    window_sign_paint,
-    nullptr
-};
-
-static void window_sign_small_mouseup(rct_window *w, rct_widgetindex widgetIndex);
-static void window_sign_small_dropdown(rct_window *w, rct_widgetindex widgetIndex, sint32 dropdownIndex);
-static void window_sign_small_invalidate(rct_window *w);
+static rct_window_event_list window_sign_events = { nullptr,
```

Comment:
Can this be changed?

Reply:
This looks ugly. It would be far better to put all entries on their own lines. That also prevents having to move the whole lot whenever this is renamed.